extensions/powershell/windowspowershell.dsc.extension.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://aka.ms/dsc/schemas/v3/bundled/resource/manifest.json",
+    "type": "Microsoft.Windows.Extension/WindowsPowerShell",
```

Comment:
I think you should call this `Microsoft.DSC.Transitional/PSDesiredStateConfiguration`